### ISSUE TITLE
spk installation operation failed

### ISSUE THREAD

**Issue by ipa64** [oznu/docker-homebridge#117]
Hello,
Because I wanted to redo a clean installation , I have deleted the HomebridgeDocker-x86_64-1.0.7.spk package from my Synology, but now it's not possible to install it again, i get this error: Operation failed

Thanks for help

**Comment by oznu** [comment]
* Remove the package from Package Center.
* Go to the DSM docker app and delete anything related to homebridge, check the images and container tabs.
* Try uploading the SPK to package centre again.

**Comment by ipa64** [comment]
- package already removed
- oznu/homebridge image deleted
- oznu/homebridge container deleted
package installation still failed

**Comment by oznu** [comment]
I'm not able to replicate this sorry.

You might have to do the install manually: https://github.com/oznu/docker-homebridge/wiki/Homebridge-on-Synology

**Comment by ipa64** [comment]
OK don't mind
That what I did
The only thing not working now is access to logs from UI!
logs are located here: /volume1/docker/homebridge/logs/homebridge.log
Tried that in platforms config file:
            "platform": "config",
            "name": "Config",
            "port": 8080,
            "log": {
            "method": "file",
            "path": "/volume1/docker/homebridge/logs/homebridge.log"
And got this error:

Loading logs from file
CMD: tail -n 200 -f /volume1/docker/homebridge/logs/homebridge.log

tail: cannot open '/volume1/docker/homebridge/logs/homebridge.log' for reading: No such file or directory
tail: no files remaining

**Comment by oznu** [comment]
Did you enable the UI using the environment variable or did you install manually?

**Comment by ipa64** [comment]
Hi, 
Like you said I had to recreate the container.
Is there some other environment variable to set ?
Do I understand well, I just have to recreate the container with HOMEBRIDGE_CONFIG_UI=1 environment variable, and at the first start it should auto install UI ?
Thx

**Comment by oznu** [comment]
Correct. Make sure you add the environment variable before starting the container for the first time.

**Comment by ipa64** [comment]
Wouaw all is working again only in few minutes !
Thank you a lot for your job and support.
Have a nice day
